package.json

```diff
@@ -66,6 +68,7 @@
     "@angular/compiler-cli": "^10.1.0-next.4",
     "@angular/core": "^10.1.0-next.4",
     "@angular/language-service": "^10.1.0-next.4",
+    "@nativescript/schematics": "^10.0.2",
```

Comment:
Do we really need that at the end? Can you try to remove this, run `npm install`, **restart the server** and try if there are any errors?

Reply:
What's the state about that?

Reply:
- Unnecessary dependency deleted. Rebuild project again and everything is working